geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeConnectionCommandJUnitTest.java

```diff
@@ -30,7 +30,7 @@
  * The GfshCommandJUnitTest class is a test suite of test cases testing the contract and
  * functionality of the GfshCommand class for implementing GemFire shell (Gfsh) commands.
  *
- * @see org.apache.geode.management.internal.cli.commands.GfshCommand
+ * @see org.apache.geode.management.cli.GfshCommand
```

Comment:
This javadoc dosen't seem to match the test class as it stands today, possibly a holdover from before the gfsh command class were refactored?

Furthermore, the class is named `*JUnitTest` but is categorized as an `IntegrationTest`

Reply:
I removed the comment as it stands not making sense.

Reply:
Also renamed the test to be `DescribeConnectionCommandIntegrationTest`